src/main/java/use_cases/PostManager.java

```diff
@@ -66,39 +66,57 @@ public void createPost(User owner, LocalDateTime postedTime, Recipe recipe, Stri
 
     /**
      * Comments a post iff could be done returns true
-     * @param post
-     * @param comment
-     * @return
+     * @param post: The post object itself
+     * @param comment: The comment is being added
+     * @return true iff it could be added the comment
      */
     public boolean commentPost(Post post, Comment comment){
         return interactPost(post, comment, true);
     }
 
     /**
      *  likes a post and saves the state
-     * @param post
-     * @param user
-     * @return
+     * @param post: the post is being liked
+     * @param user: the user that is giving the like
+     * @return true if the post could be liked
      */
     public boolean likePost(Post post, User user) {
         return interactPost(post, user, false);
     }
 
     /**
      * Adds a post directly from object
-     * @param post
+     * @param post: the post object is being added
      */
     public void createPost(Post post){
         this.posts.add(post);
     }
 
     /**
      * Return method that returns a list of the posts
-     * @return
+     * @return a list of all the posts currrently available
      */
     public ArrayList<Post> getPosts() {
         return this.posts;
     }
 
+    /**
+     * Given a post as parameter, returns the same post
+     * that is stored in the PostManager
+     * @param post: the post object
+     * @return the post stored in the postManager
+     */
+    public Post getSpecificPost(Post post)
```

Comment:
I don't think we should pass in a Post to this. Think about it - if you already have the Post object for the Post you want, why would you need to call this? PostID might make more sense, in my opinion.